_pages/mobile-interview-project.md

```diff
@@ -8,40 +8,55 @@ header:
 
 # Hello!
 
-*This site outlines the info and guidelines you’ll need to complete Square’s Mobile Take-Home Project: An Employee Directory App. You’ll have a week to complete the question – the actual engineering work should take about 3-4 hours, and you can split this across the week however you see fit (all at once, over a few days, etc). If you have any questions during the process please share them with your recruiting contact and we’ll get back to you ASAP. Good luck!*
+Thank you for choosing to interview at Square! We are so excited for you and we hope that you have a great experience with us.
+
+This site outlines the info and guidelines you’ll need to complete Square’s Mobile Take-Home Project: an employee directory app.
+
+You’ll have a week to complete the question – the actual engineering work should take about 3-4 hours, and you can split this across the week however you see fit (all at once, over a few days, etc). If you have any questions during the process, please share them with your recruiting contact and we’ll get back to you ASAP.
+
+Good luck!
 
 # Overview
-Build an employee directory app that shows a list of employees parsed from the provided JSON endpoint.
 
-Take care to properly handle errors returned from the endpoint (or other network errors like timeouts), and ensure you do not waste network bandwidth – load expensive resources such as large photos on-demand only. Photos at a given URL will never change. Once one is loaded, you do not need to reload the photo. If an employee’s photo changes, they will be given a new photo URL (eg, you can treat the URL as a GUID).
+Build an employee directory app that shows a list of employees from the provided endpoint.
+
+The app should display a list (or any kind of collection view!) which shows all the employees returned from the JSON endpoint described below. Each item in the view should contain a summary of the employee, including their photo, name, and team at minimum. You may add more information to the summary if you want, or sort employees in any fashion you’d like – sort and group by name, team, etc.
 
-All screens and views which load from the network should display proper loading, empty, and error states when content is not available. If images fail to load, displaying a placeholder is fine.
+There should be some UX to reload the employee list from within the app at any time. The UX can be done in any way you want: a button, pull-to-refresh, etc.
```

Comment:
Is this new? I don't recall this being required before. I don't recall seeing this as part of standard projects.

Reply:
Yes! please see the RFC: https://docs.google.com/document/d/1Yu1bcg9WrqsnNCIWuI38Y9QBFy8oIEYAxgBl2q4FQKM/

Reply:
Oh, interesting. Okay then, and thanks!